gammapy/image/maps.py

```diff
@@ -220,18 +220,19 @@ def write(self, filename, *args, **kwargs):
         hdu = self.to_image_hdu()
         hdu.writeto(filename, *args, **kwargs)
 
-    def coordinates(self, coord_type='skycoord', origin=0, mode='center'):
+    def pix_coordinates(self, mode='center'):
```

Comment:
Totally minor, but can you rename it to `coordinates_pix()`? This has the slight advantage, that when typing `coord...` and hitting tab completion, it will be listed in IPython.